lm-feedCore-iOS/lm-feedCore-iOS/Source/Classes/Constants/LMStringConstants.swift

```diff
@@ -19,10 +62,58 @@ public struct LMStringConstants {
     public var oKText = "OK"
     
     // CreatePost Strings
-    public var createPostNavTitle = "Create a Post"
     public var addMoreText = "Add More"
     public var addPhotoText = "Add Photo"
     public var addVideoText = "Add Video"
     public var attachFiles = "Attach Files"
     public var addPoll = "Add Poll"
+    
+    public var createPostTitle: String {
+        String(format: "Create a %@", pluralizeOrCapitalize(to: postVariable, withAction: .allSmallSingular))
+    }
+    
+    public var postDetailTitle: String {
+        pluralizeOrCapitalize(to: postVariable, withAction: .firstLetterCapitalSingular)
+    }
+    
+    public var postingInProgress: String {
+        String(format: "A %@ is already uploading!", pluralizeOrCapitalize(to: postVariable, withAction: .allSmallSingular))
+    }
+    
+    public var editPost: String {
+        String(format: "Edit %@", pluralizeOrCapitalize(to: postVariable, withAction: .firstLetterCapitalSingular))
+    }
+    
+    public var unpinThisPost: String {
+        String(format: "Unpin This %@", pluralizeOrCapitalize(to: postVariable, withAction: .firstLetterCapitalSingular))
+    }
+    public var pinThisPost: String {
+        String(format: "Pin This %@", pluralizeOrCapitalize(to: postVariable, withAction: .firstLetterCapitalSingular))
+    }
+    public var creatingResource: String {
+        String(format: "Creating %@", pluralizeOrCapitalize(to: postVariable, withAction: .firstLetterCapitalSingular))
+    }
+    public var newPost: String {
+        String(format: "NEW %@", pluralizeOrCapitalize(to: postVariable, withAction: .allCapitalSingular))
+    }
+    
+    public func reportSubtitle(isComment: Bool) -> String {
+        String(format: "You would be able to report this %@ after selecting a problem.", pluralizeOrCapitalize(to: isComment ? commentVariable : postVariable, withAction: .allSmallSingular))
+    }
+    
+    public var deletePost: String {
+        String(format: "Delete %@", pluralizeOrCapitalize(to: postVariable, withAction: .firstLetterCapitalSingular))
+    }
+    
+    public var deletePostMessage: String {
+        String(format: "Are you sure you want to delete this %@? This action cannot be reversed", pluralizeOrCapitalize(to: postVariable, withAction: .allSmallSingular))
+    }
+    
+    public var writeComment: String {
+        String(format: "Write a %@", pluralizeOrCapitalize(to: commentVariable, withAction: .allSmallSingular))
+    }
+    
+    public var noCommentPermission: String {
+        String(format: "You do not have permission to %@", pluralizeOrCapitalize(to: commentVariable, withAction: .allSmallSingular))
```

Comment:
```suggestion
        String(format: "You do not have permission to create a %@", pluralizeOrCapitalize(to: commentVariable, withAction: .allSmallSingular))
```

Reply:
Please confirm with PRD once.

Eg I renamed comments to be response (Niyo's case)

Now the above string will be -> "You do not have permission to response", Instead it should be "You do not have permission to create a response.